internal/artifacts/module_arch.go

```diff
@@ -36,14 +36,14 @@ func ExecuteBuild(source, target string, extensions []string, moduleName, platfo
 }
 
 // ExecuteSoloBuild - executes build of module from stand along command
-func ExecuteSoloBuild(source, target string, extensions []string, moduleName, platform string, wdGetter func() (string, error)) error {
+func ExecuteSoloBuild(source, target string, extensions []string, moduleName string, wdGetter func() (string, error)) error {
 	logs.Logger.Infof(buildMsg, moduleName)
 	loc, err := dir.Location(source, target, dir.Dev, extensions, wdGetter)
 	if err != nil {
 		return errors.Wrapf(err, buildFailedMsg, moduleName)
 	}
 	targetLoc := dir.ModuleLocation(loc, target)
-	err = buildModule(loc, targetLoc, moduleName, platform)
+	err = buildModule(loc, targetLoc, moduleName, "cf")
```

Comment:
Should this really always be `cf`?

Reply:
no, we should always pack, therefore I provide valid platform. I don't want to make changes to the buildModule function

Reply:
If the module is defined to be build only for `neo` for example, `packModule` will not do anything. Is this what we want?

Reply:
I see you changed it now, so never mind